pages/partials/questionFooter.ejs

```diff
@@ -74,30 +74,28 @@
       </div>
       <% } %>
     </form>
+  </div>
+<% } %>
+<script>
+    $('form').on('submit', function(e) {
```

Comment:
I'm a bit worried by the fact that this will also find other forms on the page (like the one for submitting issues). It shouldn't cause any damage, but I don't like having random JS triggering if we can avoid it.

Perhaps we could add a class like `main-question-form` to the question form and change the line above to only use `$('form.main-question-form')` or similar?

Reply:
I was wondering about this also for the "alert for unsaved" issue. It would be helpful if we have a unique identifier for question forms. Should we add an `id` instead? Or you prefer using a class for this.

Reply:
The main form already has `class="question-form"`, seems like we could use that here?

Reply:
Yeah that's what I'm currently using in my other PR (`.question-form`). Just wasn't sure since conceptually I've always treated `class` for styling and `id` for unique id'ing elements.

Reply:
Went ahead and updated to use `form.question-form`